packages/appeal-reply-service-api/data/README.md

```diff
@@ -0,0 +1,15 @@
+# Data
```

Comment:
The `/data` folder is not required - remove

Reply:
Weird, already did that. Deleted.